frontend/src/components/Admin/ModifyCourseForm.jsx

```
@@ -1,109 +1,131 @@
-          >
-            Delete
-          </button>
-        </div>
-      </form>
-    );
-};
+      )}
+
+      {/* choose another img */}
+      <div>
+        <label htlmlFor="image" className="block mb-2 text-sm font-medium text-gray-700">Update Image</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)